app/parsers/bulkrax/oai_ia_parser.rb

```diff
@@ -1,3 +1,6 @@
+# OVERRIDE: Bulkrax 4.4.1 to add collection title, noted below.
+# OVERRIDE: Bulkrax 4.4.1 to get a patch from Bulkrax 5.x.x for create_objects, noted below.
+# TODO: remove create_objects method when upgrading to Bulkrax 5.x.x
```

Comment:
@summer-cook these comments reference `#create_objects`, but I don't see any overrides for that method anymore. Does that mean these comments can be removed? Or do they need to be updated to refer to `#create_collections` now?

Reply:
@bkiahstroud yes - just removed those comments!